### ISSUE TITLE
Upload datasets on Huggingface

### ISSUE THREAD

**Issue by julien-blanchon** [satellite-image-deep-learning/datasets#10]
Hey 👋🏻, I started uploading many datasets on huggingface and packaging them with the same format: same README type, provide as much metadata as possible, parquet format with preview, provide the dataset zip on a separate branch ...

I've made a collection with my current dataset + others that are already on huggingface: https://huggingface.co/collections/blanchon/geospatial-datasets-656ddb097c934a7b3c4d4619

I may reupload some datasets already there to keep a consistent format.
If people here are interested, we can create an organisation to manage this together.

**Comment by adamjstewart** [comment]
Yes, we would love to onboard more datasets to https://huggingface.co/torchgeo! We don't yet have documentation for this, but the current approach is to nominate one of your datasets for adoption, then the technical steering committee checks things like licenses, then we move your repo to the TorchGeo org.

P.S. parquet is great for vector data but awful for raster data, you may have to convince me that parquet is a good idea...

**Comment by julien-blanchon** [comment]
Hey guys !

It's a pretty old issues haha.

I think parquets is great even outside of the vector data world, including image, audio ... It's use everywhere on huggingface.
I agree it's not ideal for doing lazy tiling/chunking, which is typically the case in geo. But for "ready to use" dataset (ie, already preprocess ...) it's not a big deal. My initial idea was to provide already preprocess dataset that will integrate with huggingface `load_dataset` so they can be load with ease

**Comment by adamjstewart** [comment]
Never used huggingface `load_dataset` before, all of our TorchGeo datasets have custom data loaders. More work to write, but supports a lot more geospatial use cases.
